api/src/main/java/org/jenkinsci/plugins/workflow/graph/FlowNode.java

```diff
@@ -141,6 +141,20 @@ public String getDisplayName() {
         else            return getTypeDisplayName();
```

Comment:
Well, `getDisplayName()` is overriding `ModelObject.getDisplayName()`, it's hard to say that it's not actually being used somewhere when accessing the flow node as a model object. Do you think it is safe to deprecate it?